app/src/main/res/values/strings.xml

```diff
@@ -157,7 +156,7 @@
     <string name="max_volume">Max volume</string>
     <string name="max_weight">Max weight</string>
     <string name="history_measurement">HISTORY</string>
-    <string name="application_warning_text">Due to service errors we will be closing\nthe application in exactly:</string>
+    <string name="application_warning_text">Due to service errors we will be closing the application in exactly:</string>
```

Comment:
Side comment: To reiterate on the doomsday counter - not killing the app and resetting it to its initial state would be significantly better UX. Which will in turn significantly decrease user uninstall events.